src/require_files.rb

```diff
@@ -0,0 +1,31 @@
+require 'i18n'
+require 'yaml'
+
+require_relative 'help_methods/input'
+require_relative 'help_methods/output/sign_in_output'
+require_relative 'help_methods/output/registration_output'
+require_relative 'help_methods/output/withdraw_output'
+require_relative 'help_methods/output/send_money_output'
+require_relative 'help_methods/output/puts_money_output'
+require_relative 'help_methods/output/output'
+require_relative 'config/i18n'
+require_relative 'account/account_connect'
+require_relative 'account/account'
+require_relative 'account/account_registration'
+require_relative 'account/account_login'
+require_relative 'cards/card'
+require_relative 'cards/virtual_card'
+require_relative 'cards/capitalist_card'
+require_relative 'cards/usual_card'
+require_relative 'operations/card_operations'
+require_relative 'operations/help_operations'
+require_relative 'operations/money_operations'
+require_relative 'storage/storage'
+require_relative 'validation/validation'
+require_relative 'views/help_view'
+require_relative 'views/destroy_account_view'
+require_relative 'views/card_view'
+require_relative 'views/withdraw_money_view'
+require_relative 'views/put_money_view'
+require_relative 'views/send_money_view'
+require_relative 'views/view'
```

Comment:
move all `require` and `require_relative` here

Reply:
And rename the file

Reply:
renamed